tools/enso4igv/src/main/resources/org/enso/tools/enso4igv/enso.tmLanguage.json

```diff
@@ -85,6 +85,16 @@
 						"matches": "\\\""
 					}]
 				},
+				{
+					"name": "string.quoted.triple.begin",
+                                        "begin": "^(\\s*)\"\"\"",
+                                        "end": "^(?!\\1\\s+)(?!\\s*$)"
+				},
+				{
+					"name": "string.quoted.triple.middle",
+                                        "begin": "^(\\s*)[^\\s].*\"\"\"",
+                                        "end": "^(?!\\1\\s+)(?!\\s*$)"
+				},
```

Comment:
Note that we got tabs and spaces mixed in this file
![image](https://github.com/enso-org/enso/assets/1436948/9741c418-79bf-4054-a9a3-e968a40b168a)

Let's make it consistent, to whatever but to be consistent as otherwise it looks weird.

Reply:
Also, I think apart from `"""` we also should add `'''`, as apparently Enso also supports triple quotes for this.

```
from Standard.Base import IO

x = """
       a
       b
y = """c
       d
       e

main =
    IO.println x
    IO.println y
```
works and happily prints
```
a
b
c
d
e
```

Reply:
Done in 59430dc